g2p_seq2seq/g2p.py

```diff
@@ -364,18 +423,48 @@ def interactive(self):
         break
       if not word:
         break
-      print(self.decode_word(word))
-
+      if(self.mode and self.mode == 'g2p'):
+        print(self.decode_word(word))
+      else:
+        print(self.decode_pronunciation(word.strip().split()))
+  
+  
+  def calc_edit_distance(self, s1, s2):
```

Comment:
Keep phoneme error rate feature in a separate pull request please.

Reply:
I will keep PER in a separate PR. For this PR, should I not calculate PER at all ?

Reply:
Done!